pkg/process/metadata/parser/nodejs/nodejs.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package nodejs wraps functions to guess service name for node applications
+package nodejs
+
+import (
+	"encoding/json"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"os"
+	"path"
+	"path/filepath"
```

Comment:
nit:

in the repo we're sorting and separating the imports to groups

then

```suggestion
	"encoding/json"
	"os"
	"path"
	"path/filepath"

	"github.com/DataDog/datadog-agent/pkg/util/log"
```